packages/app/ui/components/MessageInput/MessageInputBase.tsx

```diff
@@ -201,6 +206,7 @@ export const MessageInputContainer = memo(
               </View>
             ) : (
               <View marginBottom="$xs">
+                {showWayfindingTooltip && <Notices.ChatInputTooltip />}
```

Comment:
are we putting it here for positioning reasons? seems like it could maybe be placed in ChatInput where we initially call the hook.

Reply:
yeah, clearer position targeting. I can try to refactor if you think it's meaningfully detrimental?